packages/ketcher-react/src/script/ui/component/view/Atom/Atom.jsx

```diff
@@ -16,19 +16,42 @@
 
 import classes from './Atom.module.less'
 import clsx from 'clsx'
+import { ElementColor } from 'ketcher-core'
+import styled from '@emotion/styled'
+import α from 'color-alpha'
 
 function Atom({ el, shortcut, selected, ...props }) {
+  const atomColor = ElementColor[el.label] || '#000'
+  const AtomButton = styled('button')`
+    color: ${atomColor};
+    border-color: ${atomColor};
+    &:hover {
+      background-color: ${α(atomColor, 0.2)};
+    }
+    &:active {
+      background-color: ${α(atomColor, 0.8)};
+    }
+    &.selected {
+      color: #fff;
+      background-color: ${α(atomColor, 0.8)};
+
+      &:hover {
+        background-color: ${atomColor};
+      }
+    }
+  `
```

Comment:
In order to prevent recreating AtomButton function, we can move it outside the Atom component (e.g., create getAtomButton function, which accepts atomColor ant returns styled component)